src/Command/DigitalPostSendCommand.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\DigitalPost;
+use App\Entity\Document;
+use App\Repository\DigitalPostRepository;
+use App\Service\DigitalPostHelper;
+use App\Service\DocumentUploader;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:digital-post:send',
```

Comment:
We usually name commands `tvist1:...` although i missed it in `MailTemplateRenderCommand`.